frontend/packages/topology/src/filters/KindFilterDropdown.tsx

```diff
@@ -74,7 +74,12 @@ const KindFilterDropdown: React.FC<KindFilterDropdownProps> = ({
         </Button>
       </span>
       {kindFilters.map((filter) => (
-        <SelectOption key={filter.id} value={filter.id} isChecked={filter.value}>
+        <SelectOption
+          key={filter.id}
+          value={filter.id}
+          isChecked={filter.value}
+          data-test={filter.labelKey ? t(filter.labelKey) : filter.label}
```

Comment:
I think we don't want and need to translate `data-test` attributes.

```suggestion
          data-test={filter.label}
```

Reply:
Updated